kura/org.eclipse.kura.easse/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,23 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: OSGi Event Admin Server Sent Events
+Bundle-SymbolicName: org.eclipse.kura.easse;singleton:=true
+Bundle-Version: 1.0.0.qualifier
+Bundle-Vendor: Eclipse Kura
+Bundle-Category: Kura OSGi Event Admin SSE
+Import-Package: com.eclipsesource.json;version="0.9.4",
+ javax.servlet;version="3.1.0",
+ javax.servlet.http;version="3.1.0",
+ org.eclipse.kura.util.base;version="[1.0,2.0)",
+ org.eclipse.kura.util.collection;version="[1.0,2.0)",
+ org.osgi.framework;version="[1.7.0,2.0.0)",
+ org.osgi.service.event;version="1.3.0",
+ org.osgi.service.http;version="1.2.0",
+ org.osgi.util.tracker;version="1.5.0",
+ org.slf4j;version="1.6.4"
+Export-Package: org.eclipse.kura.easse;version="1.0.0"
+Bundle-ClassPath: .,
+ lib/jetty-eventsource-servlet-1.0.1-SNAPSHOT.jar,
+ lib/jetty-continuation-9.4.0.v20161208.jar
```

Comment:
Currently our target platform already has org.eclipse.jetty.continuation_9.3.9.v20160517. Is there a particular requirement for jetty-continuation-9.4.0?